youtube_dl/extractor/volat.py

```diff
@@ -0,0 +1,41 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class VolAtIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?vol\.at/[^?#]*?/(?P<id>[0-9]+)'
```

Comment:
Should not accept `http://www.vol.at//5593454` or `http://www.vol.at/blue/man/group/5593454`.